python/tvm/relay/backend/compile_engine.py

```diff
@@ -253,7 +255,16 @@ def select_implementation(op, attrs, inputs, out_type, target, use_autotvm=True)
             and msg not in autotvm.task.DispatchContext.warning_messages
         ):
             autotvm.task.DispatchContext.warning_messages.add(msg)
-            autotvm_logger.warning(msg)
+            if autotvm_logger.level == logging.DEBUG:  # only print if in debug mode
+                autotvm_logger.warning(msg)
```

Comment:
Can this just be `autotvm_logger.debug(msg)` so that we can simplify this logic to
```
if _first_warning:
  # print the first time warning.
autotvm_logger.debug(msg)
```
I believe it's fine to have the first time warning in both cases. Advance developers won't be mad to see one more warning.